lib/view/splashscreen.dart

```diff
@@ -9,16 +11,23 @@ class SpalshScreen extends StatefulWidget {
 }
 
 class _SpalshScreenState extends State<SpalshScreen> {
+  InlineImageProvider cacheImageProvider =
+      GetIt.instance<InlineImageProvider>();
   @override
   void initState() {
-    // TODO: implement initState
     super.initState();
+    _startImageCaching();
+  }
+
+  Future<void> _startImageCaching() async {
```

Comment:
We wanted to remove the splash screen. Do we want to still want to process the images here?